content/get-started/using-github/troubleshooting-connectivity-problems.md

```diff
@@ -32,6 +32,8 @@ If you're unable to verify with the captcha:
 
 Switching from cloning via SSH to cloning via HTTPS, or vice versa may improve connectivity. For more information, see "[Troubleshooting cloning errors](/repositories/creating-and-managing-repositories/troubleshooting-cloning-errors)."
 
+An alternative method is to use SSH over port 443, see "[Using SSH over the HTTPS port](/authentication/troubleshooting-ssh/using-ssh-over-the-https-port)"
```

Comment:
Suggested change:

> If you prefer to use SSH but the port is blocked, see (your link) for an alternative port.

Reply:
I like the suggested phrasing! Updated in 49606ee3848ae1cc64528ad4847aad3b9afc8680.